client/python/armada_client/client.py

```diff
@@ -194,3 +195,34 @@ def unwatch_events(event_stream) -> None:
         :return: nothing
         """
         event_stream.cancel()
+
+    def create_job_request(
+        self, queue: str, job_set_id: str, job_request_items: List, **job_params
+    ):
+        """Create a job request.
+
+        :param queue: The name of the queue
+        :param job_set_id: The name of the job set (a grouping of jobs)
+        :param job_params: Job Object
```

Comment:
I'm confused by job_request_items and **job_params.  

Should we have both?

Reply:
Yes, because job_request_items are a list of `submit_pb2.JobSubmitRequestItem` where as `job_params` covers all other possible kwargs. Can you think of a better way to word it? 

I'll let you respond, but you have made me realise a docstring param is missing so I will add that in when we have finished this discussion.